.circleci/config.yml

```diff
@@ -57,7 +57,7 @@ executors:
     docker:
       - image: ministryofjustice/cloud-platform-tools:2.1
         environment:
-          GITHUB_TEAM_NAME_SLUG: laa-estimate-eligibility
+          GITHUB_TEAM_NAME_SLUG: laa-check-client-qualifies
```

Comment:
I think this one should be like in CODEOWNERS and reference the team name - which doesn't have an "laa-" prefix